### PR TITLE
Support ed25519 host keys, used in OpenSSH 6.5+

### PR DESCRIPTION
**Pull request by gordonmessmer**

### CODE REVIEW

File: src/lib/Bcfg2/Server/Plugins/SSHbase.py [hunk 1]
```diff
@@ -421,7 +427,7 @@ def build_hk(self, entry, metadata):
 
     def GenerateHostKeyPair(self, client, filename):
         """Generate new host key pair for client."""
-        match = re.search(r'(ssh_host_(?:((?:ecd|d|r)sa)_)?key)', filename)
+        match = re.search(r'(ssh_host_(?:((?:ecd|d|r)sa|ed25519)_)?key)', filename)
```
Comment (AlexanderS) [inline src/lib/Bcfg2/Server/Plugins/SSHbase.py]: Line too long :-(

**Comment by solj** [comment]
👍

**Comment by AlexanderS** [comment]
:+1:

**Comment by gordonmessmer** [comment]
Ping

**Comment by solj** [comment]
Merged, thanks!

**Pull request merged**